lib/workers/branch/lock-files.js

```diff
@@ -161,6 +189,28 @@ async function writeExistingFiles(config) {
   }
 }
 
+function listLocalLibs(dependencies) {
+  logger.trace(`listLocalLibs (${dependencies})`);
+  const toCopy = [];
+  for (const libName in dependencies) {
+    if ({}.hasOwnProperty.call(dependencies, libName)) {
+      const libVersion = dependencies[libName];
+      if (libVersion.startsWith('file:')) {
+        if (libVersion.endsWith('.tgz')) {
+          logger.error(
+            `Link to local file '${
+              libVersion
+            }' is not supported. Please do it like: 'file:/path/to/folder'`
+          );
+        } else {
+          toCopy.push(libVersion.substring(5));
```

Comment:
I prefer not to use "magic numbers". instead, so something like `.substring('file:'.length);`

Reply:
Yes, good point